dev/src/index.ts

```diff
@@ -749,6 +750,18 @@ export class Firestore {
     return document.build();
   }
 
+  /**
+   * Creates a new `BundleBuilder` instance to package selected Firestore data into
+   * a bundle.
+   *
+   * @param bundleId. The id of the bundle. When loaded on clients, client SDKs use this id
+   * and the timestamp associated with the built bundle to tell if it has been loaded already.
+   * If not specified, a random string will be used.
+   */
+  bundle(name?: string): BundleBuilder {
```

Comment:
Should this be `bundleBuilder()`?

Reply:
This is an outcome of the API review, the argument here is in Node `Builder` is seldom used.